core/ui_mixins/quick_panel.py

```diff
@@ -500,3 +500,30 @@ def on_selection(self, commit):
     def on_selection_async(self, commit):
         sublime.active_window().run_command("hide_panel", {"panel": "output.show_commit_info"})
         self.on_done(commit)
+
+
+def show_stash_panel(entries, on_done):
```

Comment:
I think is it better to have this function than specifying everything each time.  Maybe StashPanel 
 https://github.com/divmain/GitSavvy/pull/744/files#diff-6809e629d1384bcdee2aa618a5425cbfR517  is an overkill. I just copied the above one and removed all we don't need.